utils/cuda_to_sycl/converter/README.md

```diff
@@ -1,14 +1,16 @@
-# The SYCL Convertor
+# The SYCL Converter
 
-A `*.cu` file is a combination of Host CUDA API and device CUDA kernel.
+This utility enables developers with existing CUDA kernel code to convert this to SYCL kernel code.
 
-This SYCL patterhn is able to re-use an existing CUDA kernel from a `*.cu` file into a C++ kernel, following these steps:
+A `*.cu` file is a combination of the Host CUDA API and device CUDA kernel code.
 
-1. For the device kernel, we will introduce a SYCL functor that will encapsulate 
+The SYCL pattern described in this document is able to accommodate existing CUDA kernel code from a `*.cu` file using the following steps:
```

Comment:
I don't really understand this. Can we improve it? Is it saying something like `Existing CUDA kernel code can be transformed into SYCL code using the pattern described in this document?` While we're at it, it would be nice to get 80 chars per line :)